conftest.py

```diff
@@ -0,0 +1,3 @@
+# In this file you can put any configuration for pytest.
```

Comment:
I found that the existence of this file is necessary for pytest to run correctly. 😭